File: codigos-fonte/fibonacci.js
//Fibonacci para JavaScript

function fibonacci(termos){
	
	const valores = {
		a: 0,
		b: 1,
		ac: 0
	};

	for (idx = 0; idx < termos; idx++){
		console.log(valores.a);
		valores.ac = valores.a + valores.b;
		valores.a = valores.b;
		valores.b = valores.ac;
	}
	return
}

fibonacci(64)

File: codigos-fonte/fibonacci.c
//Fibonacci para C

#include <stdio.h>

struct elementos {	
	//Certamente há maneiras mais elegantes de representar um inteiro de 64 bits.
	long long int a;
	long long int b;
	long long int ac;
};

int fibonacci(int termos){
	struct elementos valores = {0, 1, 0};

	for (int idx = 0; idx < termos; idx++){
		printf("%lli\n", valores.a);
		valores.ac = valores.a + valores.b;
		valores.a = valores.b;
		valores.b = valores.ac;
	}
	return 0;
}

int main(){
	int result = fibonacci(64);
	return 0;
}
